### PR TITLE
Revert local StorageClass

### PR DESCRIPTION
**Pull request by SStorm**
## Summary of the changes / Why this is an improvement

This reverts the change that introduced the local StorageClass. The SC seems to be causing issues in Azure, and the reasoning why it was introduced is unclear.

## Checklist

 - [x] [CLA](https://crate.io/community/contribute/cla/) is signed

### CODE REVIEW

**Review by lukasbals — APPROVED** [review]
Looks good to me 👍🏽

File: tests/test_bootstrap.py [hunk 1]
```diff
@@ -109,7 +109,7 @@ async def test_bootstrap_license(
                         "secretKeyRef": {"key": "license", "name": f"license-{name}"},
                     },
                     "name": "my-crate-cluster",
-                    "version": "4.1.5",
+                    "version": "4.3.3",
```
Comment (turbo-ele) [inline tests/test_bootstrap.py:112]: note: `4.3.4` would be the latest `4.3.x` release, while `4.4.0` is the latest stable

Comment (SStorm) [reply tests/test_bootstrap.py:112]: Good point, I will change it to 4.4.0 in a subsequent PR if that's ok.

Comment (lukasbals) [reply tests/test_bootstrap.py:112]: Haha true. I just approved a few minutes before `4.4.0` was out. 😂

**Pull request merged**